src/modules/iotjs_module_i2c.c

```diff
@@ -79,11 +79,11 @@ iotjs_i2c_t* iotjs_i2c_instance_from_reqwrap(iotjs_i2c_reqwrap_t* i2c_reqwrap) {
 
 static void iotjs_i2c_destroy(iotjs_i2c_t* i2c) {
   IOTJS_VALIDATED_STRUCT_DESTRUCTOR(iotjs_i2c_t, i2c);
-  i2c_destroy_platform_data(_this->platform_data);
+  iotjs_i2c_destroy_platform_data(_this->platform_data);
   IOTJS_RELEASE(i2c);
 }
 
-static void i2c_worker(uv_work_t* work_req) {
+static void iotjs_i2c_worker(uv_work_t* work_req) {
```

Comment:
@LaszloLango We used `i2c_worker` here instead of `iotjs_i2c_worker` because it is a static function, and prefix `iotjs_` is used for global names.

Reply:
There are some inconsistency here. Other static functions that use the `iotjs_` prefix in this file.
```
static iotjs_i2c_t* iotjs_i2c_create
static iotjs_i2c_reqwrap_t* iotjs_i2c_reqwrap_create
static void iotjs_i2c_reqwrap_destroy
static void iotjs_i2c_destroy
```

Other modules also uses `iotjs_` prefix for those functions (and also for `_worker` functions).
```
static void iotjs_gpio_write_worker
static void iotjs_gpio_read_worker
static void iotjs_gpio_close_worker
static void iotjs_gpio_after_worker
static void iotjs_uart_write_worker
static void iotjs_uart_close_worker
static void iotjs_uart_after_worker
static void iotjs_spi_transfer_worker
static void iotjs_spi_close_worker
static void iotjs_pwm_common_worker
static void iotjs_pwm_after_worker
static void iotjs_adc_read_worker
static void iotjs_adc_close_worker
```

Which one should we use here and in other modules? I think we should use the same coding conversion and we have to touch these function while we are working on #1367, so it is a good opportunity to do it now. :)

Reply:
@LaszloLango I found the inconsistencies also. I think it is good opportunity to change. Could you use `iotjs_` for global names only during wokring on #1367? If you find other inconsistencies, I would be happy for removing inconsistent style.